client/my-sites/subscribers/components/jetpack-empty-list-view/jetpack-empty-list-view.tsx

```diff
@@ -0,0 +1,131 @@
+import { useTranslate, default as i18n } from 'i18n-calypso';
+import { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+import './style.scss';
+
+type EmptyListCTALinkProps = {
+	icon: JSX.Element;
+	text: string;
+	onClick: () => void;
+};
+
+const EmptyListCTALink = ( { icon, text, onClick }: EmptyListCTALinkProps ) => {
+	return (
+		<Card
+			className="jetpack-empty-list-view__cta-link"
+			size="small"
+			as="button"
+			onClick={ onClick }
+		>
+			<CardBody className="jetpack-empty-list-view__card-body">
+				<Icon className="jetpack-empty-list-view__cta-link-icon" icon={ icon } size={ 20 } />
+				<span className="jetpack-empty-list-view__cta-link-text">{ text }</span>
+				<Icon
+					className="jetpack-empty-list-view__cta-link-icon"
+					icon={ chevronRight }
+					size={ 20 }
+				/>
+			</CardBody>
+		</Card>
+	);
+};
+
+const JetpackEmptyListView = () => {
+	const translate = useTranslate();
+	const selectedSite = useSelector( getSelectedSite );
+
+	// Record an event when the empty view is rendered
+	useEffect( () => {
+		recordTracksEvent( 'calypso_subscribers_empty_view_displayed' );
+	}, [] );
+
+	const handleMethodSelect = ( method: string ) => {
+		recordTracksEvent( 'calypso_subscribers_empty_view_add_method_clicked', {
+			method,
+		} );
+		if ( method === 'substack' ) {
+			if ( isJetpackCloud() ) {
+				window.location.href = `https://wordpress.com/import/newsletter/substack/${
+					selectedSite?.slug || selectedSite?.ID || ''
+				}`;
+			} else {
+				page( `/import/newsletter/substack/${ selectedSite?.slug || selectedSite?.ID || '' }` );
+			}
+			return;
+		}
+		// Update URL hash with selected method
+		window.location.hash = `#add-subscribers?method=${ method }`;
+	};
+
+	return (
+		<div className="jetpack-empty-list-view">
+			<h2 className="jetpack-empty-list-view__title">
+				{ translate( 'Add subscribers to %s', {
+					args: [ selectedSite?.title || '' ],
+					comment: "%s is the site's title",
+				} ) }
+			</h2>
+			<p className="jetpack-empty-list-view__description">
+				{ i18n.fixMe( {
+					text: translate( 'No subscribers yet? Turn your site visitors into subscribers.' ),
+					newCopy: translate(
+						'No subscribers yet? {{howToTurnVisitorsLink}}Turn your site visitors into subscribers.{{/howToTurnVisitorsLink}}',
+						{
+							components: {
+								howToTurnVisitorsLink: (
+									<a
+										href={ localizeUrl(
+											'https://jetpack.com/support/jetpack-blocks/subscription-form-block/'
+										) }
+										target="_blank"
+										rel="noopener noreferrer"
+										onClick={ () =>
+											recordTracksEvent( 'calypso_subscribers_empty_view_subscribe_block_clicked' )
+										}
+									/>
+								),
+							},
+						}
+					),
+					oldCopy: '',
+				} ) }
```

Comment:
Do we need the `fixMe` call here? It is meant to only render `newCopy` if `text` has translations, otherwise render `oldCopy`.

So at the very least there is a bit of confusion I think. Ideally, `text` should be the string used for the `newCopy` translate call, so `"No subscribers yet? {{howToTurnVisitorsLink}}Turn your site visitors into subscribers.{{/howToTurnVisitorsLink}}"` in this case 🤔

Reply:
Ok - I think I see where this is going. Basically we want to render nothing (`''`) if translation doesn't exist. I think that may not be a bad use case, but you would render the empty string in that case (perhaps we could update `oldCopy` to also accept `null` and pass `null` there 🤔 

My second comment still holds though, that `text` is the thing we check for translations...

Reply:
> Yes, I think I hijacked the intended use there. 😅 null would be great for new strings. This way I don't have to wait and remember to come back and ship the PR when translations are ready.

Yeah, nice and smart 😄 

We could certainly update to accept `null`